### PR TITLE
Drop component.io support

### PR DESCRIPTION
**Pull request by ronkorving**
This drops support for component.io. Reasoning:

- component.io needs to die in a fire (sorry to be so blunt, but the pain...).
- the build systems I personally wanna support depend on NPM and can either deal with `require('events')` themselves (like browserify), or can pick up the `events` module via package.json (like webpack).
- having the try/catch block in the code as it was before causes require-analysis-based build systems to break (since they can't resolve the dependency that's inside the try-block).

So adding the events module as a dependency to package.json should solve this.

Note: minor/major version bump

### CODE REVIEW

**Comment by AlmirKadric** [comment]
LGTM 👍 
HALLELUJAH 🙇

**Comment by ronkorving** [comment]
Please run a test in your environment and let me know if really LGTM :)

File: package.json [hunk 1]
```diff
@@ -1,31 +1,45 @@
...
+      "name": "Ron Korving",
+      "email": "rkorving@wizcorp.jp"
     }
+  ],
+  "scripts": {
+    "test": "node test"
+  },
+  "main": "index.js",
+  "dependencies": {
+    "events": "1.1.1"
```
Comment (ronkorving) [inline package.json]: This was added for web builders (eg. Webpack)

**Comment by AlmirKadric** [comment]
tested with mage-templates repo and working file 👍

**Pull request merged**